coalib/bearlib/languages/documentation/DocumentationComment.py

```diff
@@ -141,3 +141,66 @@ def _parse_documentation_with_symbols(self, param_identifiers,
                 parsed.append(self.Description(desc=desc))
 
         return parsed
+
+    @classmethod
+    def from_metadata(cls, doccomment, docstyle_definition,
+                      marker, indent, range):
+        r"""
+        Assembles a list of parsed documentation comment metadata.
+
+        This function just assembles the documentation comment
+        itself, without the markers and indentation.
+
+        >>> from coalib.bearlib.languages.documentation.DocumentationComment \
+        ...     import DocumentationComment
+        >>> from coalib.bearlib.languages.documentation.DocstyleDefinition \
+        ...     import DocstyleDefinition
+        >>> from coalib.results.TextRange import TextRange
+        >>> Description = DocumentationComment.Description
+        >>> Parameter = DocumentationComment.Parameter
+        >>> python_default = DocstyleDefinition.load("python3", "default")
+        >>> parsed_doc = [Description(desc='\nDescription\n'),
+        ...               Parameter(name='age', desc='Age\n')]
```

Comment:
maybe you want add a space before `Age`? so the assembled documentation looks a bit nicer :)